### PR TITLE
Bugfix: Allow fully numeric samples naming

### PR DESCRIPTION
**Pull request by proteinosome**
Line 99 `(match.group('sample') in samples)` will return `False` for fully numeric sample naming, because `get_samples` is a numeric list, while `re.match` output is a string. Modification here forces `get_samples` to return samples in strings.

### CODE REVIEW

File: process_sample.smk [hunk 1]
```diff
@@ -14,7 +14,7 @@ def check_for_ml_tag(bams):
     for bam in bams:
         with pysam.AlignmentFile(bam, "rb", check_sq=False) as bamfile:
             for record in bamfile:
-                if record.has_tag("Ml"):
+                if record.has_tag("Ml") or record.has_tag("ML"):
```
Comment (williamrowell) [inline process_sample.smk:17]: Please revert this change.

**Review by williamrowell — CHANGES_REQUESTED** [review]
Great catch!  Thanks for submitting a PR.  Can you revert the change to `process_sample.smk`, and we'll merge this into the `develop` branch.

File: process_sample.smk (hunk 1 above)
Comment (proteinosome) [reply process_sample.smk:17]: @williamrowell Aaron made that commit. I told him about the ML tag issue, but we didn't notice you already had another pull request to detect from header instead.

Comment (williamrowell) [reply process_sample.smk:17]: Reverted the `ML` tag change in `main`. We don't commit code changes directly to main.
